docs/guide.md

```diff
@@ -0,0 +1,434 @@
+
+3. Set the slippage tolerance (the percentage of the total swap value you can tolerate losing due to the difference in actual price and the price at confirmation time). To do this, expand `Slippage tolerance` section and choose the percentage. For example, here we've chosen `0.5%`:
+
+   ![](./img/swap/swap-slippage.png)
+
+   Note that the lesser the tolerance, the greater the chance that the transaction will fail. Dex will warn you about this.
+
+   ![](./img/swap/swap-slippage-fail.png)
+
+4. Expand `Transaction Details` to view the transaction details such as the price impact or the minimum amount of tokens you will receive.
+  
+   ![](./img/swap/swap-details.png)
+
+5. If you agree to the conditions of the transaction, press `Swap` and confirm the swap.
+
+   <!-- confirmation screenshot-->
+
+### Issues
+
+You can face the following issues during the swap:
+
+- Insufficient amount of tokens.
+  
+  If the amount of tokens you want ot swap is higher than what you have in your wallet, you won't be able to perform the swap.
+
+  ![](./img/swap/swap-insufficient-tokens.png)
+
+- Not enough KLAY for transaction fee.
+  
+  If you don't have enough KLAY tokens in your wallet, you will see this error after pressing `Swap`. You won't be able to confirm the transaction unless you have enough KLAY to pay the fee.
+
+- Route between the pair of tokens is not found.
+  
+  Enable [multi-hops](#multi-hops) in the swap preferences to be able to swap a pair of tokens that doesn't have a liquidity pool.
+
+### Swap Preferences
+
+If you press the settings icon, you will see swap settings (enabling multi-hops and expert mode):
+
+![](./img/swap/swap-preferences.png)
+
+#### Multi-hops
+
+If you enable multi-hops, you will be able to swap tokens that don't have a [liquidity pool](#liquidity-pool) for their pair.
+
+Note the differences in route for two following swap operations:
+
+- If multi-hops are disabled, there is no route between `VEN` and `DEX`:
+
+  ![](./img/swap/swap-route-not-found.png)
+
+- If multi-hops are enables, there is a route between `VEN` and `DEX` through multiple other pairs:
+
+  ![](./img/swap/swap-multi-hop.png)
+
+#### Expert mode
+
+In expert mode, you only need to confirm access to the token once. After that it will be available for all future operations.
+
+## LP Provision
+
+Liquidity pools are smart contracts that hold balances of two unique tokens and enforce rules on depositing and withdrawing them.
+
+Liquidity pools contain LP tokens. When you add liquidity to a liquidity pool, you receive LP tokens. When you remove liquidity, you remove LP tokens from the pool.
+
+You can add and remove liquidity in the `Trade`>`Liquidity` tab:
+
+- If you haven't joined any pool, you will only be able to add liquidity:
+
+  ![](./img/liquidity/liquidity.png)
+
+- If you already joined pools, you will see their information as well: 
+
+  ![](./img/liquidity/liquidity-2.png)
+
+### Add liquidity
```

Comment:
Please extend it for the case when there is no pool. Initial pool or pool does not exist, so the rate is not calculated automatically, it is set during the first supply for non-existed pool